.gitignore

```diff
@@ -27,6 +27,9 @@
 .build/
 .fseventsd
 Carthage/Build
+Carthage/Checkouts
+Cartfile
```

Comment:
@alan-hadyk Do we stop using Carthage? 
cc: @igroomgrim

Reply:
@nuxzero We still use it in some project and above just the rule for ignoring `Cartfile` and `Checkouts` folder.

Reply:
I see. I just wondering why this PR ignore `Cartfile`. 🙂

Reply:
@nuxzero I had a lot of files when I've used Carthage - so I didn't want to commit them. Removed from `.gitignore`